src/Model/Subsite.php

```diff
@@ -870,7 +870,7 @@ public function domain()
         }
 
         // If there are no objects, default to the current hostname
-        return $_SERVER['HTTP_HOST'];
+        return isset($_SERVER['HTTP_HOST']) ? $_SERVER['HTTP_HOST'] : 'localhost';
```

Comment:
Please use `Director::host()` instead of this ternary operation.

Reply:
Doing so would introduce many more logic paths which were previously not included. I think it makes sense to do it anyway though.